docs/api-reference/test-utils/browser-driver.md

```diff
@@ -6,115 +6,108 @@
 
 > Note: Requires Chrome version 64 or higher
 
-A Chrome Browser test automation driver class (based on the [Chrome `DevTools` protocol](https://chromedevtools.github.io/devtools-protocol/) via the [`puppeteer`](https://github.com/GoogleChrome/puppeteer) module. The `BrowserDriver` class is primarily intended for automating runs of browser based tests from shell scripts.
+A Chrome Browser test automation driver class (based on the [Chrome `DevTools` protocol](https://chromedevtools.github.io/devtools-protocol/) via the [`puppeteer`](https://github.com/GoogleChrome/puppeteer) module. The `BrowserDriver` class is primarily intended for automating browser based applications from shell scripts.
 
 A `BrowserDriver` is typically used to do the following:
-* Start a Chrome browser instance
-* Start a (webpack) dev server that builds a bundle of a JavaScript code to execute in the browser.
-* Opens a browser page with a URL that loads the bundle into the browser and runs the tests.
-* Extracts a pass/fail value from the browser back into node
-* Closes browser, server and terminates the current node script.
-* Passes the pass/fail value as exit status (0/1) back to the invoking shell.
+* Launch/close a Chromium browser instance
+* Start/stop a local web service.
+* Opens a browser page with a URL in the browser.
 
+To use this class, [puppeteer](https://www.npmjs.com/package/puppeteer) must be installed as a dev dependency.
 
 ## Usage
 
 ```js
 import {BrowserDriver} from 'probe.gl/test-utils';
-new BrowserDriver({...});
+new BrowserDriver({id: 'browser-test'});
 ```
 
 
-## Methods
+## Constructor
 
-### constructor
+```js
+const browserDriver = new BrowserDriver(opts);
+```
 
-`new BrowserDriver({...})`
+Parameters:
 
-### startBrowser
+* `opts` (Object)
+  - `id` (String) - an id for this `BrowserDriver` instance. Default `browser-driver`.
 
-`driver.startBrowser(options)`
 
-Returns a `Promise` that resolves when the browser has started.
+## Methods
 
+### startBrowser(options)
```

Comment:
Just FYI:  I am using the convention of documenting types in the `###` header, and dropping the extra line of the options object in the argument list, in favor of `options.---`

E.g:

```
### startBrowser(options : Object)
`options.url` - 
```

I find it makes things a bit more compact, less duplicated and easier to scan.

I realize, we have a mixed style and no agreed-upon direction in this matter, so also fine as is.